### ISSUE TITLE
Whats is Thumbnail described in the readme ? Sadly i can not figure out the changes on windows.

### ISSUE THREAD

**Issue by High-Flyer** [emacs-vs/line-reminder#32]
Greetings,
  I am sorry to ask this not being able to figure it out myself, What is [Display THumbnails](https://github.com/emacs-vs/line-reminder/tree/master#%EF%B8%8F-display-thumbnail)

This is how my config looks after activating that some fring bar showing at right, am unable to figure out its random not responding to saves, 
![Screenshot (53)](https://github.com/emacs-vs/line-reminder/assets/12442704/20ad0439-b1a3-4d36-a173-36e546b45496)
And seeing a strange micro line at bottom, this remaines no matter what mode or feature is turned on or off shown in following,
![Screenshot (57)](https://github.com/emacs-vs/line-reminder/assets/12442704/80f28421-604e-4bc5-aac4-5c0ec5ba4192)

Beside be kind to explain whats this linum mode and indicators difference, What understanding is Indicators are are default in current versions of emacs, where this linum mode was used once upon a time and a deprecated thing now ?
And does linum mode shows an auxillary line number line next to existing one ?
And to change color of the indicator in either indicators mode or the linum mode we have to insert a custom vertical indicator ? to illustrate it as our indicator?
When i try to insert those vertical rectangles in my config it keep asking me dou you want to keep UTF-8 default and marks those blocks as some /246 or so when i saves them. SO how can i customize if i just want to keep minimal but change the colour not customize everything ? 
Am very new to Emacs plz forgive me if i said or behaved some way which i am not supposed to.
Any input / guidance / tips will be more than welcome.

***config:***
```
(use-package line-reminder
  :config
  (add-hook 'prog-mode-hook  'line-reminder-mode)
  (add-hook 'org-mode-hook 'line-reminder-mode)
  ;; Using indicators
  (setq line-reminder-show-option 'indicators)  ; Can be set to 'indicators 'linum
  (setq line-reminder-bitmap 'filled-rectangle) ;; Customize the symbol of the fringe: (Default to 'line-reminder--default-bitmap)
  ;; (setq line-reminder-bitmap 'line-reminder--default-bitmap)
  (setq line-reminder-fringe-placed 'left-fringe) ;; If you change the fringe location by altering this variable: (Default to 'left-fringe)

  ;;Display thumbnail  
(setq line-reminder-thumbnail t)
(setq line-reminder-thumbnail-bitmap 'filled-rectangle))
```

Thank you.
Warm Regards

**Comment by High-Flyer** [comment]
Greetings,
Thank you for being kind to explain about linum.
I will try to stay away from linum as being amateur and with poor computer.

> I don't think this is an issue related to this package. 

I will try to investigate about following thank you.

> You can customize these faces to change the color.

Took note of it, appreciated.

Went through still clueless.
> See https://github.com/emacs-vs/line-reminder/issues/29#issuecomment-1811205817.

With Thumbnails I meant what it does ? After enabling it am seeing no difference.
Another green bar seeing at the right side.
Thumbnail block is right in Readme. Am sorry unable to figure out what it this or doing.
I tried to comfirm its not responding to links or the images or acting as the mini-map.
Here's the section from Readme.

> Display thumbnail
You can display thumbnail by setting:
(setq line-reminder-thumbnail t)
P.S. fringe is oppsing from variable line-reminder-fringe-placed, hence it's default to right-fringe
You can change the thumbnail bitmap by: (Defaul to line-reminder--default-thumbnail-bitmap)
(setq line-reminder-thumbnail-bitmap 'filled-rectangle)
Change thumbnail display symbol: (Default to "▐")
(setq line-reminder-thumb-modified-sign "▐"
      line-reminder-thumb-saved-sign "▐")

Warm Regards o7

**Comment by jcs090218** [comment]
> With Thumbnails I meant what it does ? After enabling it am seeing no difference.
> Another green bar seeing at the right side.

* The core shows precisely where the lines have been edited. (right)
* The thumbnail shows the edit throughout the document. (left)

<img src="https://github.com/emacs-vs/line-reminder/assets/8685505/65ba74d8-bb18-4786-8e0a-de27e6653553" width="50%"/>

**Comment by High-Flyer** [comment]
Greetings, 
Thank you mr Shen ([jcs090218](https://github.com/jcs090218)) I got everything covered, Thumbnails are like the indication for Whole buffer saves (minimap save indicator mode).

Some words of appreciation for what this package meant to me,
Coming from Notepad++ isnt easy to switch spl when the very basics we get from notepad++ or Geany or VSCodium or vim are missing to such an advance IDE / Distribution like Emacs. Its like if am not getting the basics What am gonna do there spl if not from programming background. 
I can not explain how long I was searching Reddits, Stacks, and forums to look for this (Show Recent Changes in buffer and "line-number-mode"), either they ended on `diff-hl` or `git-gutter` like package meant only to be used with version control system or some kind of `diff/ediff` code Where a minibuffer will shown with all the differences, or some ugly bright `highlight-changes-mode` block where the consequences of showing differences were hampering the overall experience dealing with editor like some fringes / auto-completion / highlight on incorrect code. I am new to emacs i dont jump to Version control now till am accomodate comfortably with Emacs. Thatswhy i told you to change the description of this package. :)
Then I just started browsing Melpa with just typed line and went through every packages shown. And found `line-reminder` :).
And it was like ohh I dont have to go back or look for something else now.
Thank you from my heart (Am sorry for my inability to support monetarily, am unemployed.)
Feel free to close this question-issue-session.

When it came to interactions with you its more than a generous philanthropist exchange.
Thank you for everything Mr. Shen the package, kind replies and contributions.
More power to you.
Warm Regards o7
